src/services/Audius/delegate.ts

```diff
@@ -37,18 +44,54 @@ export type GetPendingUndelegateRequestResponse = {
   target: Address
 }
 
-export type IncreaseDelegateStakeEvent = {
+export type GetIncreaseDelegateStakeEventsResponse = {
   blockNumber: number
   delegator: Address
   increaseAmount: BN
   serviceProvider: Address
 }
 
-export type DecreaseDelegateStakeEvent = {
+export type GetDecreaseDelegateStakeEvaluatedResponse = {
+  blockNumber: number
+  delegator: Address
+  amount: BN
+  serviceProvider: Address
+}
+
+export type GetDecreaseDelegateStakeRequestedResponse = {
   blockNumber: number
+  lockupExpiryBlock: number
   delegator: Address
-  decreaseAmount: BN
+  amount: BN
+  serviceProvider: Address
+}
+
+export type GetDecreaseDelegateStakeCancelledResponse = {
```

Comment:
should this be maybe something like DecreaseDelegateStakeCancelledEventResponse?

Reply:
+1 needs `Events`. What a long name ugh